recipe/meta.yaml

```
@@ -10,34 +10,66 @@ source:
   sha256: 099d86f7c79c51a2caaf7c8598cc56e7b0a57c11b5918ddf699e82380735b6b7
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
+  skip: true  # [py<38]
 
 requirements:
   host:
-    - python >=3.8,<4.0
+    - python 
     - hatchling
     - pip
   run:
-    - python >=3.8,<4.0
+    - python
     - faker
     - typing-extensions
```

```suggestion
    - typing-extensions >=4.6.0
```
see https://github.com/litestar-org/polyfactory/blob/v2.17.0/pyproject.toml#L43C22-L43C29